### PR TITLE
Some more modernization to tests

### PR DESCRIPTION
**Pull request by langner**
Here are some further changes to the tests, which I hope make them easier manage. Note that cclib/cclib-data#11 needs to be merged in before Travis will be happy.

### CODE REVIEW

**Comment by langner** [comment]
Comment on the renaming - I think it makes sanse to call the tests that deal with logfiles "data tests", as opposed to unit tests for parsers, bridges, etc. (added a stub for parsers in one commit).

So I put all the "data tests" in the data/ subdirectory and refactored the `testsall` code in `test_data.py`.

**Comment by langner** [comment]
Any comment on this?

**Comment by berquist** [comment]
I'm going to take a look at this tomorrow; haven't been able to give it a thorough look-over until now.

**Comment by langner** [comment]
Great! Looking forward to that. I might add a few commits later today for you then.

File: test/data/testSP.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 import numpy
 
-from testall import skipForParser
-from testall import skipForLogfile
+from skip import skipForParser
+from skip import skipForLogfile
```
Comment (berquist) [inline test/data/testSP.py]: These are giving me an `ImportError`, unless I change them to relative imports. Same with `testSPun` and `testScan`.

Comment (langner) [inline test/data/testSP.py]: > These are giving me an ImportError, unless I change them to relative imports. Same with testSPun and testScan.

Are you sure the import from `skip` is the problem? Could you show the actual error you get here after the latest commits?

Comment (langner) [inline test/data/testSP.py]: > These are giving me an ImportError, unless I change them to relative imports. Same with testSPun and testScan.
> 
> Are you sure the import from skip is the problem? Could you show the actual error you get here after the latest commits?

OK, now I see this problem pertains to Python3 only, where the importing is a bit different. This should be fixed in one of the last commits.

**Comment by langner** [comment]
I forgot to test everything in Python3, and there are some importing differences and other changes. Commit  552f1c9 takes care of these things. The code is still somewhat of a mess, but the tests should now pass and they are more comprehensive than before.

Also, the method tests are not working/organized yet.

**Comment by langner** [comment]
Like I suggested in langner/cclib#4, just not testing bridges with Travis is the easy way out. And reasonable. I think it's OK, since it's not a core part of cclib. I created #200 to work on this in the unspecified future.

**Pull request merged**